src/hopen/syntax/silly_j.cljc

```diff
@@ -0,0 +1,51 @@
+(ns hopen.syntax.silly-j
+  (:require [hopen.renderer.xf :as rxf]
+            [instaparse.core :as insta]))
+
+(declare parse)
```

Comment:
Can be avoided if the render function can be defined after the parser.

Usually, the `declare` is only used when there is a cross recursion between 2 functions defined at the root level.